sparql/sparql12/syntax-triple-terms-positive/expr-tripleterm-02.rq

```diff
@@ -0,0 +1,6 @@
+PREFIX : <http://example.com/ns#>
+
+SELECT * {
+  ?s ?p ?o .
+  BIND(<<( <<(?s ?p ?o)>> :q :z )>> AS ?t)
```

Comment:
I'm not sure TripleTerm should be able to appear in the subject position of an (expression) TripleTerm pattern.